packages/fxa-content-server/package.json

```diff
@@ -154,6 +154,8 @@
     "@testing-library/react": "^12.1.5",
     "@types/backbone": "^1.4.10",
     "@types/postcss-import": "^14",
+    "@types/react": "^17.0.14",
+    "@types/react-dom": "^17.0.9",
```

Comment:
Let's have at least the major version of the types match the major version of the libraries.  (I know that the other packages are using these versions; they shouldn't.)

Reply:
Yup that makes sense.

A bit of a tangent, but do we even need React in `fxa-content-server`? I have this PR, #15491 up to remove it, and all appears to be well.